### DISCUSSION TITLE
Dock preferences

### DISCUSSION THREAD

**Discussion by PavloMuravyov** [elementary/triage#636] [Q&A] [answered]
Are there any ways to change parameters of new Dock (not only icons size and visibility params), like it was in prev versions? Or can it be replaced with "plank" 

When installed old "plank" from terminal it didn't replace new one, there are both planks together) not good experience. I can`t found any info about how to disable new Dock

**Comment by ryonakano** [comment, marked as answer]
There are no hidden preferences for the new dock. See https://github.com/orgs/elementary/discussions/619

I haven't tested and even won't recommend but you might be able to prevent the new dock from launching automatically by replacing its executable file with a shell script which just `exit 0`

**Reply by PavloMuravyov** [reply]
I already tried some actions with io.elementary.dock. When i use "killall" dock removes from screen but it come back after some seconds. After it, i simply removed dock (the biggest error in my life), and tried to reboot system. As a result i can`t log in my system after rebooting... i also tried to fix it by adding io.elementary.dock package by Recovery Mode and terminal mode in lock screen. After all these manipulations, I kept getting a failure when trying to log in. 

Error was "Session returned error code 1" . After it i was crying))) for some minutes and reinstall all system.
